### PR TITLE
Ryna-L261.carbon_storage_USA

### PR DESCRIPTION
**Pull request by rynacui**
#727

### CODE REVIEW

**Comment by rynacui** [comment]
Thanks a lot @bpbond! This is my very last chunk :tada:

**Comment by bpbond** [comment]
Waiting on @gokuliyer review.

File: R/zchunk_L261.carbon_storage_USA.R [hunk 1]
```diff
@@ -48,131 +52,193 @@ module_gcam.usa_L261.carbon_storage_USA_DISABLED <- function(command, ...) {
...
-    #
-    #
-    # NOTE: there are 'match' calls in this code. You probably want to use left_join_error_no_match
-    # For more information, see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # NOTE: This code uses vecpaste
-    # This function can be removed; see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # NOTE: This code uses repeat_and_add_vector
-    # This function can be removed; see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # ===================================================
+    # Create a vector of FERC grid regions with non-zero storage curves
```
Comment (gokuliyer) [inline R/zchunk_L261.carbon_storage_USA.R]: It's not entirely clear to me why we're doing this and the next step. Could we add some detail in the comment?

Thanks.

File: R/zchunk_L261.carbon_storage_USA.R [hunk 2]
```diff
@@ -48,131 +52,193 @@ module_gcam.usa_L261.carbon_storage_USA_DISABLED <- function(command, ...) {
...
+    # L261.DeleteSubsector_USAC: delete onshore carbon storage subsector of carbon storage sector in the USA region
+    # NOTE: leaving the offshore here so that the USA hydrogen sector has a carbon storage market
+    L261.SubsectorShrwtFllt_C %>%
+      mutate(region = region) %>% # strip off attributes like title, etc.
+      filter(region == "USA") %>%
+      semi_join(L261.DepRsrc, by = c("subsector" = "depresource")) %>%
+      select(one_of(c(LEVEL2_DATA_NAMES[["Subsector"]]))) ->
+      L261.DeleteSubsector_USAC
+
+    # Create a vector of grid level onshare carbon storage subsector with zero storage curve
```
Comment (gokuliyer) [inline R/zchunk_L261.carbon_storage_USA.R]: Same as above. Say why we need this in the comment.

**Comment by bpbond** [comment]
Holding on @gokuliyer approval.

**Pull request merged**